NitroxServer/Communication/UdpServer.cs

```diff
@@ -156,8 +156,34 @@ private Connection GetConnection(long remoteIdentifier)
         private NetPeerConfiguration BuildNetworkConfig()
         {
             NetPeerConfiguration config = new NetPeerConfiguration("Nitrox");
-            config.Port = 11000;
-            config.MaximumConnections = 100;
+            string INIContents;
```

Comment:
You can use `File.ReadAllText(@".\config.ini")` instead of `StreamReader.ReadToEnd`. Saves 2 lines.

The UdpServer is getting messy if it also contains INI reading and parsing logic. It would be better to put an INI file reading/writing/parsing class in the `NitroxModel` project and then use it here. Though I'm ok with it to keep it like this for now because of the work required.

Reply:
Fixed this in the new commit